starknet-foundry/crates/cast/src/starknet_commands/multicall.rs

```diff
@@ -36,12 +38,29 @@ struct InvokeCall {
 #[derive(Args)]
 #[command(about = "Execute multiple calls at once", long_about = None)]
 pub struct Multicall {
-    #[clap(short = 'p', long = "path")]
-    /// Path to a .toml file containing the multi call specification
-    pub path: String,
+    #[clap(subcommand)]
+    pub command: Command,
 }
 
-pub async fn multicall(
+#[derive(Debug, Subcommand)]
+pub enum Command {
```

Comment:
We already have `Commands` defined, I think this name can be confusing. What do you think about changing it to `MulticallSubcommands`?

Reply:
Ok, NVM. Clippy doesn't like when `item name starts with its containing module's name`. But what do you think about just `Subcommands`?

Reply:
I think this should be a singular name, not a plural one. And if we go with `Subcommand`, it's going to overlap with `clap::Subcommand`.

Reply:
We already have `Commands` defined in the main.rs file

Reply:
ok, right this can be plural, I do not mind [change to plural](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/3171f96a727ee43ff11cd7390fb2b1e7c79eb979)